### PR TITLE
NEXUS-8552 - Enable ExtJS Diagnostics for 4.x layout issues

### PR DESCRIPTION
**Pull request by kellyrob99**
- hide the storage fieldset to prevent layout failure
- enable diagnostics which were useful to find and fix this otherwise obtuse problem

https://issues.sonatype.org/browse/NEXUS-8552

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ Ext.define('NX.coreui.view.repository.facet.StorageFacet', {
         xtype: 'fieldset',
         cls: 'nx-form-section',
         title: NX.I18n.get('ADMIN_REPOSITORIES_DETAILS_SETTINGS_STORAGE_FACET'),
+        hidden: true,
```
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js]: here is the 'fix' without which the layout blew up with the empty fieldset, an obscure error related to 'shrinkwrap'

**Comment by djsauble** [comment]
+1

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js (hunk 1 above)
Comment (djsauble) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js]: Good find. Is the lack of fields in this facet temporary?

**Comment by jdillon** [comment]
what are these new js classes and where did they come from?

**Comment by kellyrob99** [comment]
@jdillon these are extjs classes which add detailed logging for UI layout, as explained in the link above ^^^

**Comment by jdillon** [comment]
can we fix the bug and hten leave the addition of diag stuff for consideration later?  i'm not sure we want to get into the business of checking in specific stuff like this w/o a bit more consideration for how we'll manage it.  things like this likely make moving to extjs5 even harder, so we want to be very careful.

**Comment by kellyrob99** [comment]
@jdillon I am fine with that, will leave this PR as POC in case this comes up, as without this in place the alternative to chase down issues like this appears to be commenting out components one at a time until you find the one with the problem.

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js (hunk 1 above)
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/facet/StorageFacet.js]: Yes, at some point in the future we will want to be able to configure which backing BlobStore is used by a repository, but probably not for awhile.

**Comment by kellyrob99** [comment]
Merged the single-line fix in 604a1e7c832d1e97c8cb1f0fc3141b12be33f32d

**Comment by kellyrob99** [comment]
For posterity's sake, I checked and these files are not present in 5.x source code anyhow so @jdillon 's point is... right on point

**Pull request closed without merging**